wire/internal/wire/parse.go

```diff
@@ -89,10 +93,11 @@ type ProviderSet struct {
 	// variable.
 	VarName string
 
-	Providers []*Provider
-	Bindings  []*IfaceBinding
-	Values    []*Value
-	Imports   []*ProviderSet
+	Providers    []*Provider
+	Bindings     []*IfaceBinding
+	Values       []*Value
+	Imports      []*ProviderSet
+	InjectorArgs *InjectorArgs
```

Comment:
nit: Add a comment that this is only filled in for `wire.Build`.

Reply:
Done.